### PR TITLE
added globalEvents flags json and documentation md file

### PR DESCRIPTION
**Pull request by DetKewlDog**
Figured this out using @jgaribsin 's [`globalEvents` historical dump](https://gist.github.com/jgaribsin/eb53da585ceb838ca9a1e416e3b6ee04) a while ago but I've noticed it's still not in this repo. Thought I might as well add it myself :)

```json
// status endpoint
{
  // ...
  "globalEvents": [
    {
      "eventId": 1499087,
      "id32": 0,
      "portraitId32": 0,
      "title": "BRIEFING",
      "titleId32": 2908633975,
      "message": "...",
      "messageId32": 46181626,
      "race": 1,
      "flag": 1, // <-- This is the flag value i documented
      "assignmentId32": 2311178494,
      "effectIds": [],
      "planetIndices": []
    }
  ],
  // ...
}
```

### CODE REVIEW

**Comment by chatterchats** [comment]
Please modify jrd.tree to include the update file. I also recommend adding pages to your fork repo to test your changes to the documentation

File: .github/workflows/blank.yml
Comment (dealloc) [inline .github/workflows/blank.yml:1]: I'm not entirely a fan of the name 'blank' for the workflow file, not particularly descriptive.

File: Writerside/topics/globalEvents_flags.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Global Events Flags
+
+## dictionary
+This json contains a dictionary of key: value pairs, representing the globalEvent flag (which correlates to its type) returned from the Helldivers 2 Status API endpoint.
```
Comment (dealloc) [inline Writerside/topics/globalEvents_flags.md:4]:
```suggestion
This json contains a dictionary of key/value pairs, representing the `globalEvent` flag returned from the Helldivers 2 Status API endpoint.
```

File: Writerside/topics/globalEvents_flags.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+```json
```
Comment (dealloc) [inline Writerside/topics/globalEvents_flags.md:6]: I would avoid copying the contents of `globalEvents_flags` here as any update to that file would also need to be made here.
Preferably link to the file itself to make maintenance easier

File: Writerside/topics/globalEvents_flags.md
Comment (dealloc) [inline Writerside/topics/globalEvents_flags.md:1]: Overall documentation could use work, it reads a little choppy still.
Try moving away from the 'dictionary' style of documentation and describe what global events are, point out which property indicates the global event, and in the example I'm missing what JSON you're showing (is this a planet, dispatch, ...)

**Review by dealloc — CHANGES_REQUESTED** [review]

File: .github/workflows/blank.yml
Comment (DetKewlDog) [reply .github/workflows/blank.yml:1]: Yeah i was trying to figure out gh pages, I'll delete it

File: Writerside/topics/globalEvents_flags.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Global Events Flags
+
+## dictionary
+This json contains a dictionary of key: value pairs, representing the globalEvent flag (which correlates to its type) returned from the Helldivers 2 
+Status API.
+
+```json
+{
+	"1": "Stratagem",
+	"2": "Briefing",
```
Comment (dmpaskell22) [inline Writerside/topics/globalEvents_flags.md]: wh673-256.62
USD

**Comment by dealloc** [comment]
@DetKewlDog how comes we closed this PR?
some small changes and I feel this could've been ready to merge

**Comment by DetKewlDog** [comment]
Couldn't find the time to work on it and I didn't want it to stay open for no reason :(

**Pull request closed without merging**